helm/superset/Chart.yaml

```diff
@@ -29,7 +29,7 @@ maintainers:
   - name: craig-rueda
     email: craig@craigrueda.com
     url: https://github.com/craig-rueda
-version: 0.11.1
+version: 0.11.2
```

Comment:
The README will need to be updated with the latest version as well

Reply:
Done